models/article.js

```diff
@@ -0,0 +1,38 @@
+module.exports = (sequelize, DataTypes) => {
+  const Article = sequelize.define('Article', {
+    post: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    likes: {
+      type: DataTypes.INTEGER,
+      defaultValue: 0
+    },
+    dislikes: {
+      type: DataTypes.INTEGER,
+      defaultValue: 0
+    },
+    rating: {
+      type: DataTypes.INTEGER,
+      allowNull: true
+    },
+    category: {
+      type: DataTypes.STRING,
+      allowNull: false
+    }
+
+  });
+
+  Article.associate = (models) => {
+    Article.belongsTo(models.User, {
+      foreignKey: 'userId',
+      onDelete: 'CASCADE'
+    });
+
+    /*  Article.hasMany(models.Comment, {
```

Comment:
Why is this part commented?

Reply:
Because the Comment model is not available. The app crashes when started.

Reply:
Okay.